scrapers/FamilyTherapyXXX.yml

```diff
@@ -0,0 +1,41 @@
+name: "FamilyTherapyXXX"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - familytherapyxxx.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+    scene:
+      Title: //h1/text()
+      Date:
+        selector: //meta[@itemprop="uploadDate"]/@content
+        parseDate: 2006-01-02T15:04:05-07:00
+      Details:
+        selector: //div[@class="entry-content"]/p[1]/text()|//meta[@itemprop="description"]/@content
+        concat: ''
```

Comment:
Concat doesn't do anything here.
To work it needs a proper option eg  whitespace `'  '` 
What you use is `''` which is an empty char
If you do that you'll see that we get 2 times the title because `|` union returns results for both xpaths
On from the 2 xpaths in the union need either to be removed or made more explicit
Was there a sample scene that needed the `|` to work ?

Reply:
So for concat then, is there no way to concat the strings without a separator? Ex if the two sections are broken up mid word like `fam` and `ily` I don't want a whitespace in there, I just want them straight concatted.

There were some scenes that required the other xpath to work, I'll revisit tonight and see if I can make it more explicit.